src/autopas/molecularDynamics/ParticleClassLibrary.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <math.h>
+#include <map>
+#include <vector>
+#include "autopas/particles/Particle.h"
+
+class ParticleClassLibrary {
+ public:
+  ParticleClassLibrary(std::map<unsigned long, double> &sigma, std::map<unsigned long, double> &epsilon,
```

Comment:
remove this constructor and add a function like:
```
void addType(size_t typeID, double eps, double sigma, double mass);
```
so that the lib can be filled incrementally on the fly